content/en/users/check-in/vos/_index.md

```diff
@@ -382,6 +384,183 @@ be supported in the future.
 
   Same as the update but requires setting the membership status to `Deleted`
 
+### API v2
+
+Check-in provides a REST API that allows clients to manage membership
+information only for the VOs they are authoritative for.
+
+Features:
+
+- Members of the VO are identified via their EGI Check-in Community User Identifier (CUID)
+- Membership can be limited to a specified period
+- All [REFEDS](https://wiki.refeds.org/display/STAN/eduPerson+2020-01#eduPerson202001-eduPersonAffiliation) membership affiliations are supported
+- Role titles are supported
+- Different membership status values are supported, namely `Active`, `Expired`,
+  `Deleted`, `Suspended`
+- Check-in automatically changes the membership status from `Active` to
+  `Expired` beyond the validity period
+
+#### Connection Parameters
+
+| | Production environment  | Demo environment                 | Development environment |
+|-| ----------------------- | ------------------------------- | -------------------- |
+| **VO API Base URL** | `https://aai.egi.eu/registry/co_person_roles` | `https://aai-demo.egi.eu/registry/co_person_roles` | `https://aai-dev.egi.eu/registry/co_person_roles` |
+| **CO ID**           | The number part of the API username prefix. e.g. for the username `co_2.test`, the CO ID is `2` | `***` | `***` |
```

Comment:
Can you try to use a tabpane for this?
Would be nicer to read and maintain than with such a wide table.
See https://docs.egi.eu/about/contributing/shortcodes/#code-in-multiple-languages

Reply:
Done